app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt

```diff
@@ -54,11 +54,14 @@ class MessageActivity : AppCompatActivity() {
 
         send.setOnClickListener {
             // Easter egg
+           try{
             if (content.text.isEmpty()) {
                 presenter.sendBitmap(this, BitmapFactory.decodeResource(resources, R.drawable.mix2))
             } else {
                 presenter.sendMessage(this, convertToDeviceDataModel())
-            }
+            }}catch (e: Exception){
```

Comment:
Please follow the java style guideline. Or you can run `./gradlew spotlessApply` to fix it and build will pass.